src/backend/models/pageOrder.ts

```diff
@@ -75,6 +75,28 @@ class PageOrder {
     return Promise.all(docs.map(doc => new PageOrder(doc)));
   }
 
+  /**
+   * Returns only root page's order
+   *
+   * @returns {Promise<PageOrder[]>}
+   */
+  public static async getRootPageOrder(): Promise<PageOrder> {
+    const docs = await this.getAll();
+
+    return docs.filter(doc => doc.page === '0')[0];
```

Comment:
Maybe it will be better to filter documents while querying  to database instead of using filter method? you can also use `findOne` method if you need to find only one document

Reply:
Fixed